digitalutsc/islandora_solution_pack_oralhistories
# Uls replace file get contents - addresses Issue 65

# What does this Pull Request do?
This reworks a line of code that used file_get_contents to fetch an object's transcript datastream - so that it calls directly to fedora using the islandora functions rather than rely on an http request of the web server.

# What's new?
The code change should not be noticed, but on systems that have a web firewall that prevent http calls to same server (to protect against bot attacks).  This code change will allow these kind of configurations to actually display the transcript.

# How should this be tested?
Ensure that an oral history object with a transcript can be viewed and played as before.  It may be beyond the scope of this issue to set up an environment where the web server can not make http requests from itself -- and test that with the old code to see that a transcript is not rendered, and test with the updated code to see that it fixes that issue... but that is why we needed this code fix.

# Additional Notes:
If anything, it could make the loading time of an oral history object a tiny bit faster.  This change should not effect anything else and does not require any change to documentation.
